core/primitives/src/test_utils.rs

```diff
@@ -104,6 +104,55 @@ impl SignedTransaction {
         )
     }
 
+    pub fn create_contract(
+        nonce: Nonce,
+        originator: AccountId,
+        new_account_id: AccountId,
+        code: Vec<u8>,
+        amount: Balance,
+        public_key: PublicKey,
+        signer: &dyn Signer,
+        block_hash: CryptoHash,
+    ) -> Self {
```

Comment:
Just for the record, we discussed that having these many positional arguments is hard to read on the usage site, and error-prone, but it works for now, so it is good to go.

Reply:
yeah